app/syncinfo.go

```diff
@@ -0,0 +1,84 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	replicaClient "github.com/openebs/jiva/replica/client"
+	"github.com/openebs/jiva/types"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+func SyncInfoCmd() cli.Command {
+	return cli.Command{
+		Name:      "syncinfo",
+		ShortName: "ls",
+		Action: func(c *cli.Context) {
+			if err := getSyncInfo(c); err != nil {
+				logrus.Fatalf("Error running ls command: %v", err)
+			}
+		},
+	}
+}
+
+func getSyncInfo(c *cli.Context) error {
+	controllerClient := getCli(c)
+
+	reps, err := controllerClient.ListReplicas()
+	if err != nil {
+		return err
+	}
+
+	var info *types.SyncInfo
+	format := "%v\t%v\t%v\t%v\n"
```

Comment:
You may need to add one more '\t' for snapshot column as snapshots can be bigger

Reply:
tabwriter package will automatically adjust padding as set.